### PR TITLE
Improved IP address fetch & display

### LINKED ISSUE #23: Fetch and display the IPv4/IPv6 address on the verbose

### PR DESCRIPTION
**Pull request by shivangswain**
Added a simple & better method to fetch and display public IP address in the server verbose

Closes #23

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -20,6 +20,16 @@ def obtntime():
...
 
 
+def ipaddress(v):
+    url = "https://api64.ipify.org"
+    if v == 4:
+        url = "https://api.ipify.org"
+    elif v == 6:
+        url = "https://api6.ipify.org"
+    response = urllib3.PoolManager().request('GET', url)
+    return response.data.decode('UTF-8')
```
Comment (t0xic0der) [inline main.py:30]: Nice work done with fetching distinct addresses.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
You would want to make the lines L#117 and L#121, the part of the `ipaddress` function so as to avoid repeating the same parts of code. Also, please make use of a try-except block in the function to address the cases when the server providing those addresses is unreachable.

(Don't worry about the specificity of the exception. Just catch one and let the user know that the addresses could not be fetched.)

File: main.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import asyncio, websockets, sys, click, time, os
+import asyncio, websockets, sys, click, urllib3, time
```
Comment (t0xic0der) [inline main.py:1]: `urllib3` is a not-so elegant module but lightweight - something which is perfect for our usecase. Thanks for replacing `requests` in favour of this module.

**Review by t0xic0der — APPROVED** [review]

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Just make changes listed on https://github.com/t0xic0der/sanctuary-zero/pull/87#pullrequestreview-519979051.

**Comment by shivangswain** [comment]
Just pushed a new, even simpler and faster implementation

**Pull request merged**